src/rviz/default_plugin/marker_utils.cpp

```diff
@@ -279,7 +279,7 @@ void checkQuaternion(const visualization_msgs::Marker& marker, std::stringstream
   {
     addSeparatorIfRequired(ss);
     ss << "Unnormalized quaternion in marker message.";
-    increaseWarningLevel(StatusProperty::Error, level);
+    increaseWarningLevel(StatusProperty::Warn, level);
```

Comment:
:+1: 

minor point:
A few lines above, the warning states `Uninitialized quaternion assuming identity`, but whether that's actually the case depends on the module calling the function and is not determined here.
Are the quaternions implicitly normalized when they are not already?
It might make sense to add this to the warning message too then.

Reply:
I reduced the severity level to a warning, because rviz doesn't crash with unnormalized quaternions. Errors will always cause the corresponding marker to be dropped, which shouldn't happen in this case.